triple-reads-backend/index.ts

```diff
@@ -1,12 +1,14 @@
 import express, { Express, Request, Response } from 'express';
 import dotenv from 'dotenv';
 import axios from 'axios';
+import querystring from 'querystring';
 
 dotenv.config();
 
 const app: Express = express();
 const port = process.env.PORT || 3000;
 const fusekiUrl = process.env.FUSEKI_URL || 'http://localhost:3030/triple-reads/sparql';
```

Comment:
Could we create another const with the base url ('http://localhost:3030/triple-reads')?

Reply:
?

Reply:
It is added, its below that one, on line 12

Reply:
But it is not used in line 10 and 11
```suggestion
const fusekiUrl = process.env.FUSEKI_URL || fusekiBaseUrl + '/sparql';
```